### DISCUSSION TITLE
Can we record client ip in nfs ganesha log?

### DISCUSSION THREAD

**Discussion by microyahoo** [rook/rook#12500] [General]
Can we record client ip in nfs ganesha log? For example, the time of nfs client connection and disconnection, etc., for log auditing. Can it be done by setting parameters of ganesha?  Thanks.
@BlaineEXE @thotz @gaord

**Comment by microyahoo** [comment]
nfs ganesha could record client ip in debug log like below, so we can update loglevel in cephnfs CRD.
![20230711-170518](https://github.com/rook/rook/assets/10202906/82ca4f00-34fc-4477-9867-92d4cc6d3e7e)

https://github.com/nfs-ganesha/nfs-ganesha/blob/850c6fbb942200d60d8eabed4714f29db22474f6/src/MainNFSD/nfs_worker_thread.c#L1068
https://rook.io/docs/rook/latest/CRDs/ceph-nfs-crd/#nfs-settings

**Comment by thotz** [comment]
I didn't any such option in ganesha.conf, so I don't know whether in what all logs ganesha provides client IP information
